scripts/includes/common.sh

```diff
@@ -100,23 +106,25 @@ generate_random_passwords() {
 
 configure_st2_user () {
   # Create an SSH system user (default `stanley` user may be already created)
-  if (! id stanley 2>/dev/null); then
-    sudo useradd stanley
+  if (! id $USERNAME 2>/dev/null); then
+    sudo useradd $USERNAME
   fi
 
-  sudo mkdir -p /home/stanley/.ssh
+  HOME=$(get_home ${USERNAME})
+
+  sudo mkdir -p ${HOME}/.ssh
 
   # Generate ssh keys on StackStorm box and copy over public key into remote box.
-  sudo ssh-keygen -f /home/stanley/.ssh/stanley_rsa -P ""
+  sudo ssh-keygen -f ${HOME}/.ssh/${USERNAME}_rsa -P ""
 
   # Authorize key-base access
-  sudo sh -c 'cat /home/stanley/.ssh/stanley_rsa.pub >> /home/stanley/.ssh/authorized_keys'
-  sudo chmod 0600 /home/stanley/.ssh/authorized_keys
-  sudo chmod 0700 /home/stanley/.ssh
-  sudo chown -R stanley:stanley /home/stanley
+  sudo sh -c "cat ${HOME}/.ssh/${USERNAME}_rsa.pub >> ${HOME}/.ssh/authorized_keys"
```

Comment:
I would prefer to be more explicit and call this variable `STANLEY_HOME` or similar (becase `$HOME` really represents home of a user which is running the script which is usually not stanley.

Reply:
I will make this change.